diff --git a/ulib/FStar.Printf.fst b/ulib/FStar.Printf.fst
index adc8da0036..9938da8784 100644
--- a/ulib/FStar.Printf.fst
+++ b/ulib/FStar.Printf.fst
@@ -195,8 +195,15 @@ let no_extensions : extension_parser = fun s -> None
 inline_for_extraction
 let sprintf
     (s:string{normalize_term (b2t (Some? (parse_format_string s no_extensions)))})
-    : normalize_term (dir_type (Some?.v (parse_format_string s no_extensions)))
-    = normalize_term (string_of_dirs (Some?.v (parse_format_string s no_extensions)) (fun s -> s))
+    : normalize_term (dir_type (Some?.v (
+      let x: (x: option (list dir)) = parse_format_string s no_extensions in
+      // admitP (Some? x);
+      match x with
+      | Some x -> Some x
+      | _ -> Some (magic ())
+    )
+    ))
+    = admit (); normalize_term (string_of_dirs (Some?.v (parse_format_string s no_extensions)) (fun s -> s))
 
 
 /// `ext_sprintf`: An extensible version of sprintf
